ios/FluentUI/Core/Theme/FluentTheme.swift

```diff
@@ -42,12 +42,19 @@ public class FluentTheme: NSObject, ObservableObject {
             return FontInfo(name: font.fontName, size: font.pointSize)
         })
 
-        let colorTokenSet = TokenSet<ColorToken, UIColor>(FluentTheme.defaultColors(_:), colorOverrides)
-        let shadowTokenSet = TokenSet<ShadowToken, ShadowInfo>(FluentTheme.defaultShadows(_:), shadowOverrides)
+#if os(visionOS)
+        // We have custom overrides for `defaultColors` in visionOS.
+        let defaultColorFunction = FluentTheme.defaultColor_visionOS(_:)
+#else
```

Comment:
Should this be an else if for iOS? A bit of an alarm goes off for things where iOS is the "default" and visionOS is an override as some rigor there now may make it easier and more straightforward for someone standing up the next platform to know where they need work (runtime work turns into compile-time work)

Reply:
So at more of a code level, feels suspicious that there is no analogous +iOS.swift file or defaultColor_iOS func

Reply:
I think your points are very valid, but they seem to come from a position of assuming that visionOS and iOS are peers in the repo. At the moment, that's not the case; we are 100% intentionally treating visionOS as an offshoot/variation of iOS for the time being.

When/if visionOS is a [fully-supported Fluent 2 platform, with all the requisite design and documentation that that entails](https://fluent2.microsoft.design/components/ios), then we should reconsider this division. Until then, it feels appropriate for the "light" support for the OS in the system.

Reply:
One other thing to consider is the whole "no light/dark mode" is a uniquely visionOS thing (and watchOS but I don't see us supporting fluent on watchOS). Maybe the next platform will also follow this and I'll eat these words but I would expect any new platforms to want to use the iOS tokens then.